src/diffusers/pipelines/ltx/pipeline_ltx_video2video.py

```diff
@@ -0,0 +1,860 @@
+        latents_std = latents_std.view(1, -1, 1, 1, 1).to(latents.device, latents.dtype)
+        latents = latents * latents_std / scaling_factor + latents_mean
+        return latents
+
+    def prepare_latents(
+        self,
+        video: Optional[torch.Tensor] = None,
+        batch_size: int = 1,
+        num_channels_latents: int = 128,
+        height: int = 512,
+        width: int = 704,
+        num_frames: int = 161,
+        dtype: Optional[torch.dtype] = None,
+        device: Optional[torch.device] = None,
+        generator: Optional[torch.Generator] = None,
+        latents: Optional[torch.Tensor] = None,
+        sigma: torch.Tensor = 1.0,
+    ) -> torch.Tensor:
+        # TODO: do we need the `conditioning_mask` here? I think `conditioning_mask` should be all ones.
+        height = height // self.vae_spatial_compression_ratio
+        width = width // self.vae_spatial_compression_ratio
+
+        # TODO: should video_processor take care of it? Because for Cog, we get a 5D tensor here.
+        # `video` memory layout is (num_frames, num_channels, height, width)
+        if video.ndim == 4:
+            video = video.unsqueeze(0)
+
+        num_frames = (
+            (num_frames - 1) // self.vae_temporal_compression_ratio + 1 if latents is None else latents.size(2)
+        )
+        shape = (batch_size, num_channels_latents, num_frames, height, width)
+        mask_shape = (batch_size, 1, num_frames, height, width)
+
+        if latents is not None:
+            conditioning_mask = latents.new_ones(shape)
+            conditioning_mask = self._pack_latents(
+                conditioning_mask, self.transformer_spatial_patch_size, self.transformer_temporal_patch_size
+            )
+            return latents.to(device=device, dtype=dtype), conditioning_mask
+
+        if isinstance(generator, list):
+            if len(generator) != batch_size:
+                raise ValueError(
+                    f"You have passed a list of generators of length {len(generator)}, but requested an effective batch"
+                    f" size of {batch_size}. Make sure the batch size matches the length of the generators."
+                )
+
+            init_latents = [
+                retrieve_latents(self.vae.encode(video[i].unsqueeze(0).permute(0, 2, 1, 3, 4)), generator[i])
+                for i in range(batch_size)
+            ]
+        else: # `premute()` because we want `batch_size, num_channels, num_frames, height, width`
+            init_latents = [
+                retrieve_latents(self.vae.encode(vid.unsqueeze(0).permute(0, 2, 1, 3, 4)), generator) for vid in video
+            ]
+
+        init_latents = torch.cat(init_latents, dim=0).to(dtype)
+        init_latents = self._normalize_latents(init_latents, self.vae.latents_mean, self.vae.latents_std)
+        # `ones()` as we want to condition on all?
+        conditioning_mask = torch.ones(mask_shape, device=device, dtype=dtype)
```

Comment:
`conditioning_mask` will not be required anywhere in the video-to-video pipeline I think. It was only required in image-to-video because we wanted first latent frame to be actual image latent, and all the other latent frames to be pure noise. Probably did not require it as a simple `torch.cat` would have sufficed, but it is what it is now haha